src/main/java/HomePage.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class HomePage {
+
+    WebDriver driver;
+
+    public HomePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public HomePage searchNintendoSwitch() {
+        By InputSearch = By.id("twotabsearchtextbox");
```

Comment:
Can you move the selectors as class member? outside of the function. All the selectos.